.github/pull_request_template.md

```diff
@@ -5,7 +5,33 @@ Describe in plain language the motivation (bug, feature, etc.) behind the change
 Fixes: #NNN, #NNN, ...
 
 ## Release notes
+<!--
 
-If the PR changes the user experience, write a short summary of the changes. See the [CONTRIBUTING](https://github.com/vectorizedio/redpanda/blob/dev/CONTRIBUTING.md) guidelines for details.
+If this PR does not need to be included in the release notes,
+then simply have a bullet point for `none` directly under
+the `Release notes` section, e.g.
 
-Release note: [1-2 sentences of what this PR changes]
+* none
+
+Otherwise, change the `### Todo` section to one or more of the following sections
+below. A section must have at least 1 bullet point. You can add multiple
+sections with multiple bullet points if this PR represents multiple release
+note items. See the CONTRIBUTING.md guidelines for more details.
+
+### Features
+
+* Short description of the feature. Explain how to configure the new feature if applicable.
+
+### Improvements
+
+* Short description of how this PR improves redpanda.
+
+### Bug fixes
+
+* https://github.com/vectorizedio/redpanda/issues/ISSUE_NUM Short description of the fixed bug prefixed by a link to the relevant GitHub issue if applicable.
```

Comment:
Can we skip copy-pasting the bug description in here?  Since we include issue links, that's something the release notes generator should be able to scrape itself.

Reply:
Just realised we also still have the `Fixes:` line above, so I don't think we necessarily need this section at all.  The release notes generator should validate that at least one of Fixes, Features or Improvements is populated.